doc/metrics.md

```diff
@@ -29,26 +29,26 @@ To view the raw data that LXD collects, use the [`lxc query`](lxc_query.md) comm
 ```{terminal}
 :input: lxc query /1.0/metrics
 
-# HELP lxd_cpu_seconds_total The total number of CPU time used in seconds.
-# TYPE lxd_cpu_seconds_total counter
-lxd_cpu_seconds_total{cpu="0",mode="system",name="u1",project="default",type="container"} 60.304517
```

Comment:
what happened to these examples?

Reply:
Now the first metrics that are shown in the raw output are the API rates metrics and the metrics in the previous example will be shown below, so I just updated the example output to better fit what the user will see when querying the metrics.

Reply:
mmm, thats fine, but we've lost the examples for the non api requests metrics too.

Reply:
I used a bigger example that includes non API requests metrics.